openedx/features/calendar/models.py

```diff
@@ -0,0 +1,20 @@
+"""
+Models for Calendar
+"""
+
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class UserCalendar(models.Model):
```

Comment:
Should this have a uniqueness constraint of `(user, course_key)`?

Reply:
I think so, yes. Otherwise a user could have multiple calendar configs for the same course.